tests/_compilertestrunner.livecodescript

```diff
@@ -0,0 +1,637 @@
+﻿script "CompilerTestRunner"
+/*
+Copyright (C) 2015 LiveCode Ltd.
+
+This file is part of LiveCode.
+
+LiveCode is free software; you can redistribute it and/or modify it under
+the terms of the GNU General Public License v3 as published by the Free
+Software Foundation.
+
+LiveCode is distributed in the hope that it will be useful, but WITHOUT ANY
+WARRANTY; without even the implied warranty of MERCHANTABILITY or
+FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License
+for more details.
+
+You should have received a copy of the GNU General Public License
+along with LiveCode.  If not see <http://www.gnu.org/licenses/>.  */
+
+-- FIXME provide this on the command line
+constant kLogFilename = "_test_suite.log"
+
+on startup
+   try
+      if $LC_COMPILE is empty then
+         throw "LC_COMPILE environment var must be set to lc-compile to use"
```

Comment:
I think it makes sense to throw this error somewhere in `CompilerTestRunnerMain` at the point where you know whether or not you need `$LC_COMPILE` to be set for this invocation (for example, using the `help` command doesn't require `$LC_COMPILE` to be set in the environment).